libs/platform/experience/src/services/experience/static-data/default-experience-static.service.ts

```diff
@@ -0,0 +1,40 @@
+import { inject } from '@spryker-oryx/di';
+import { Component } from '../models';
+import { ExperienceStaticData, ExperienceStaticService } from './static-data';
+
+export class DefaultExperienceStaticService implements ExperienceStaticService {
```

Comment:
I'm not against this service, but I don't really like we have to iterate through all components recursively twice, because we moved autoId here 🤔 
(and without autoid functionality, overhead with separate service is probably not justified)